tutorial/source/working_memory.ipynb

```diff
@@ -127,16 +127,21 @@
    "outputs": [],
    "source": [
     "from contextlib import ExitStack\n",
-    "from pyro.contrib.util import iter_plates_to_shape, rexpand\n",
+    "from pyro.contrib.util import rexpand\n",
     "\n",
     "def multiround_model(l):\n",
-    "    # The shape of l is (batch dims..., number of rounds)\n",
+    "    # The shape of l is (*batch dims, number of rounds)\n",
+    "    batch_dims = l.shape[:-1]\n",
     "    with ExitStack() as stack:\n",
-    "        for plate in iter_plates_to_shape(l.shape[:-1]):\n",
-    "            stack.enter_context(plate)\n",
+    "        for i, dim in enumerate(batch_dims[::-1]):\n",
```

Comment:
nit: could you call this variable `size` rather than `dim`? (`pyro.plate` has a different arg named `dim` and it is confusing to pass `dim` to the `size` arg)

Reply:
sure thing